ocaml/quicktest/quicktest.ml

```diff
@@ -857,7 +859,7 @@ let _ =
     "-iso-sr-path", Arg.String (fun x -> Quicktest_storage.iso_path := x), "Path to ISO SR";
     "-single", Arg.String (fun x -> tests_to_run := [ x ]), Printf.sprintf "Only run one test (possibilities are %s)" (String.concat ", " all_tests) ;
     "-all", Arg.Unit (fun () -> tests_to_run := all_tests), Printf.sprintf "Run all tests (%s)" (String.concat ", " all_tests);
-    "-only-sr", Arg.String (fun x -> Quicktest_storage.only_sr_name := Some x), "Run tests only on SR with specified name";
+    "-default-sr", Arg.Unit (fun () -> use_default_sr := true), "Only run SR tests on the pool's default SR";
```

Comment:
@edwintorok do we want to actually *remove* the `-only-sr` flag, or could we have both?

Reply:
I was under the impression we want to remove it.

Reply:
-only-sr was added as a convenience flag while testing GFS2. Its just as convenient to change the default SR in Xencenter as passing the name, so I would not want to have two mechanisms if we do not need two.